### DISCUSSION TITLE
v4.3.1

### DISCUSSION THREAD

**Discussion by elkan1788** [hugo-next/hugo-theme-next#62] [Announcements]
此次修补版本发布主要是做了少许的优化，参考如下：

- 升级 Waline 至最新版本支持文章表情互动（点赞功能）
- 与配置文件联动，优化深浅色主题的切换逻辑
- 给文章导航样式增加树形连线效果
- 增加主题的模板功能，参考[hugo-theme-next-starter](https://github.com/hugo-next/hugo-theme-next-starter)

欢迎大家使用和反馈，祝你好运！！！ :tada: :tada: :tada:

<hr /><em>This discussion was created from the release <a href='https://github.com/hugo-next/hugo-theme-next/releases/tag/v4.3.1'>v4.3.1</a>.</em>

**Comment by MuXiongGuo** [comment]
请问现在可以使用mermaid 吗  为什么文章里设置了true, 配置文件也设置了开启 还是没效果 感谢

**Reply by MuXiongGuo** [reply]
翻阅了 之前的release 发现 已经支持了🙏
